lua/telescope/actions/set.lua

```diff
@@ -148,11 +148,19 @@ end
 ---@param direction number: The direction of the scrolling
 --      Valid directions include: "1", "-1"
 action_set.scroll_previewer = function (prompt_bufnr, direction)
+  local picker = action_state.get_current_picker(prompt_bufnr)
```

Comment:
Hmm i think i prefer `local previewer = action_state.get_current_picker(prompt_bufnr)` makes the things later a little bit more clear.

Otherwise LGTM thanks :)

Reply:
Got your point, but previewer.previewer looks strange. I went with current_picker.

Reply:
Oh sorry i meant to only get the previewer because we never use the current picker.
`local previewer = action_state.get_current_picker(prompt_bufnr).previewer`

Reply:
But tbh its fine like this as well

Reply:
That makes great sense, should have seen that when I was writing `previewer.previewer`.
Thanks a lot.

Reply:
no worries, is was kinda my bad :laughing: